features/step_definitions/networking.rb

```diff
@@ -1122,4 +1122,40 @@
     step %Q/the outputs should contain "sctp"/
     $i +=1
   end
-end
\ No newline at end of file
+end
+
+Given /^get egress data file directory to the#{OPT_SYM} clipboard$/ do | cb_name |
+  ensure_admin_tagged
+  cb_name = "cb_egress_directory" unless cb_name
+  @result = admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
```

Comment:
@huiran0826 I would recommend adding the class network.operator into lib/openshift/ so we can get the information in a more object-oriented way.

Reply:
There is a class NetworkOperator in https://github.com/openshift/verification-tests/pull/1252/files, after that PR merged, I can update these codes to use that class.

Reply:
@pruan-rht Updated it, thanks!